src/wp-admin/includes/class-wp-privacy-requests-table.php

```diff
@@ -109,7 +109,8 @@ protected function get_default_primary_column_name() {
 	protected function get_request_counts() {
 		global $wpdb;
 
-		$cache_key = $this->post_type . '-' . $this->request_type;
+		// This duplicates wp_count_posts() with an additional post_name clause.
+		$cache_key = wp_cache_get_last_changed( 'posts' ) . '-' . $this->post_type . '-' . $this->request_type;
```

Comment:
```suggestion
		$cache_key = wp_cache_get_last_changed( 'posts' ) . ":{$this->post_type}:$this->request_type}";
```

:1234: Switch to `:` as separator applies throughout, it's the most common separator in existing cache keys. At a guess because it's not used in slugs so reduces the change of collisions.

Quote type is a nitpick.

Reply:
Updated :)